src/app/qgsvectorlayerproperties.cpp

```diff
@@ -249,6 +249,12 @@ QgsVectorLayerProperties::QgsVectorLayerProperties(
   diagLayout->addWidget( diagramPropertiesDialog );
   mDiagramFrame->setLayout( diagLayout );
 
+  // WMS Name as layer short name
+  mLayerShortNameLineEdit->setText( layer->shortName() );
+  // WMS Name validator
+  QRegExp shortNameRx( "^[A-Za-z][A-Za-z0-9\._-]*" );
```

Comment:
This QRegExp is used only once in the QgsVectorLayerProperties, but the same is used in QgsRasterLayerProperties, QgsProjectProperties and in QgsGroupWMSDataDialog.
Do I have to do it a static member for the 4 classes ?